src/math-translator.js

```diff
@@ -15,16 +15,21 @@ const MATH_RULES_LOCALES = {
     NO_THOUSAND_SEP: ['ko', 'ps', 'ka'],
     DECIMAL_COMMA: ['cs', 'fr', 'de', 'pl', 'bg', 'nb', 'tr', 'da', 'sr', 'lol',
             'ro', 'nl', 'hu', 'az', 'it', 'pt', 'pt-pt', 'sv', 'el', 'id', 'ka',
-            'ru'],
+            'ru', 'ta'],
     // TODO(danielhollas):remove 'bg' from TIMES_AS_CDOT
     // when \mathbin{.} becomes available for them
-    TIMES_AS_CDOT: ['cs', 'pl', 'de', 'nb', 'sr', 'ro', 'hu', 'sv', 'da', 'bg',
-            'lol'],
-    DIV_AS_COLON: ['cs', 'de', 'bg', 'hu', 'uk', 'da', 'hy', 'pl', 'lol', 'id',
+    TIMES_AS_CDOT: ['cs', 'pl', 'de', 'nb', 'sr', 'ro', 'hu', 'sv', 'da', 'bg'],
+    CDOT_AS_TIMES: ['fr', 'ps', 'pt-pt', 'ta'],
```

Comment:
This rule was missing because I previously thought that US strings only contain times.
This is not strictly related to this PR, but I included it so that I can update all locale rules accordingly and have a complete treatment for multiplication notation.

Reply:
Not a direct comment on \\times and \\cdot, but was thinking about alternate structures like \\div and \\frac{}{} (Would that be more complex since it's not simply symbol replacement? Just pure intellectual curiosity, not actual need.)

Reply:
Yeah, this would be doable albeit a bit harder to handle all possible corner cases. But I think for this particular case it's not needed because at least in the primary school, div and frac are different concepts, so you wouldn't want to mix them up.